packages/itinerary-body/src/place-row.js

```diff
@@ -126,15 +138,6 @@ const PlaceRow = ({
             ))}
         </Styled.PlaceDetails>
       </Styled.DetailsColumn>
-      {showMapButtonColumn && (
```

Comment:
This changes the layout so that the MapButton will be confined within the DetailsColumn. I think this should be reverted. See proposed way of doing things here: https://github.com/ibi-group/otp-ui/pull/5.